modules/Search/components/Facet.tsx

```diff
@@ -71,7 +74,12 @@ function Facet({ attribute, items, refine }: RefinementListProvided & Refinement
                     <Disclosure.Panel className={styles.panel}>
                         <ul className={styles.list}>
                             {visibleItems.map((item) => (
-                                <li key={`${attribute}_${item.label}`} className={styles.listItem}>
+                                // eslint-disable-next-line
+                                <li
+                                    key={`${attribute}_${item.label}`}
+                                    className={styles.listItem}
+                                    onClick={() => refine(item.value)} // make entire list item clickable
```

Comment:
✖️ 
If you wanted to make the entire element clickable, it's better to not add the `onClick` listener, but rather update styles so that `listItem` doesn't have any padding, and all styles should come from `listItemInner`. That way you will make the whole element clickable with just CSS

Reply:
Sure, making the update!

Reply:
Updated!